test/test_grammars.rb

```diff
@@ -3,12 +3,16 @@
 class TestGrammars < Minitest::Test
   ROOT = File.expand_path("../..", __FILE__)
 
-  # These grammars have no license but have been grandfathered in. New grammars
-  # must have a license that allows redistribution.
-  UNLICENSED_GRAMMARS_WHITELIST = %w[
-    vendor/grammars/Sublime-Lasso
-    vendor/grammars/Sublime-REBOL
-    vendor/grammars/x86-assembly-textmate-bundle
+  LICENSE_WHITELIST = [
+    # This grammar's MIT license is inside a subdirectory.
+    "vendor/grammars/SublimePapyrus",
+    "vendor/grammars/gap-tmbundle",
```

Comment:
Could you move `gap-tmbundle` into its own paragraph so it's clearer the subdirectory comment doesn't apply to it? Maybe a comment like "This grammar has a nonstandard but acceptable license" would be helpful too.